docs/understand/model.md

```diff
@@ -18,77 +18,69 @@ defines the common concepts of lakeFS.
 
 ## Objects
 
-lakeFS is an object store and borrows concepts from S3.
-
-An _object store_ links objects to paths. An _object_ holds:
-
-* Some _contents_, with unlimited size and format.
-* Some _metadata_, including
-  + _size_ in bytes,
-  + the _creation time_, a timestamp with seconds resolution,
-  + a _checksum_ string which uniquely identifies the contents,
-  + some _user metadata_, a small map of strings to strings.
-
-Similarly to many object stores, lakeFS objects are immutable and never rewritten. They can
-be entirely replaced or deleted, but not modified.
+lakeFS is an interface to manage objects in an object store.
 
 The actual data itself is not stored inside lakeFS directly but in an [underlying object store](#concepts-unique-to-lakefs).
-lakeFS will manage these writes and store a pointer to the object in its metadata database.
+lakeFS manages pointers and additional metadata about these objects. 
 {: .note }
 
 ## Version Control
 
-lakeFS borrows its concepts for version control from Git.
+lakeFS introduces version control semantics for data. Most of these will be familiar to [Git](https://git-scm.com/){:target="_blank"} users:
 
 ### Repository
 
-In lakeFS, a _repository_ is a logical namespace used to group together objects, branches, and commits.
-It can be considered the lakeFS analog of a bucket in an object store. Since it has version control qualities, it's also analogous to a repository in Git.
+In lakeFS, a _repository_ is a set of related objects (or collections of objects). In many cases, these represent tables of [various formats](https://lakefs.io/hudi-iceberg-and-delta-lake-data-lake-table-formats-compared/){:target="_blank"} for tabular data, or a set of unstructured objects such as images, videos, sensor data, etc.
+
+lakeFS represents repositories as a logical namespace used to group together objects, branches, and commits - analogous to a repository in Git.
 
 ### Commits
-Commits are immutable "checkpoints" containing an entire snapshot of a repository at a given point in time.
-This is very similar to commits in Git. Each commit contains metadata - the committer, timestamp, a commit message, as well as arbitrary key/value pairs you can choose to add.
-Using commits, you can view your Data Lake at a certain point in its history and you're guaranteed that the data you see is exactly is it was at the point of committing it.
 
-Every repository has exactly one _initial commit_ with no parents. A commit with more than one parent is
-a _merge commit_. Currently lakeFS only supports merge commits with two parents.
+Using commits, you can view a [repository](#repository) at a certain point in its history and you're guaranteed that the data you see is exactly as it was at the point of committing it.
 
-#### Identifying commits
-{: .no_toc }
+These commits are immutable "checkpoints" containing all contents of a repository at a given point in time. This is very similar to commits in Git.
```

Comment:
At this point, I will stay away from git reference, as it should be clear by now. So just stick to what a commit is and how to identify it:

Commits are ...
You can identify since they ...

Reply:
Good calls, removed